src/components/CurrencyDropDown/CurrencyDropDown.js

```diff
@@ -1,14 +1,63 @@
-const CurrencyDropDown = (props) => {
-  return (
-    <div>
-      <select name="currency" id="currency">
-        <option value="USD">USD</option>
-        <option value="EUR">EUR</option>
-        <option value="JPY">JPY</option>
-        <option value="GBP">GBP</option>
-      </select>
-    </div>
-  );
-};
+import React from 'react';
+import axios from 'axios';
+
+export default class CurrencyDropDown extends React.Component {
+  state = {
+    currencies: null,
+    isLoading: false,
+    hasError: false,
+    currencyName: '',
+    currencyRate: 0 
+  }
+
+  getCurrencyChange = (value) => {
+    let currency = '';
+    if (value === 'USD') {
+      currency = this.state.currencies.rates.usd.name;
+    } else if (value === 'EUR') {
+      currency = this.state.currencies.rates.eur.name;
+
+    } else if (value === 'GBP') {
+      currency = this.state.currencies.rates.gbp.name;
+    }
```

Comment:
const currency =  this.state.currencies.rates[value.toLowerCase()].name;

Reply:
or you could have the value coming in lowercase so then you can:
const currency = this.state.currencies.rates[value].name;